### PR TITLE
move include files to <saffire/*.h> (ref #217)

### LINKED ISSUE #217: move include files to <saffire/*.h>
move include files to <saffire/*.h>

This makes it a bit easier (and cleaner) to distinguish between third party header files (libxml, stdio etc) and saffire include headers..

### PR DESCRIPTION
**Pull request by ghost**
Fix #217

### CODE REVIEW

File: include/saffire/gitversion.h [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Warning: this file is autogenerated. Do not modify.
```
Comment (jaytaph) [inline include/saffire/gitversion.h]: I think we should remove this file from the repository and place it in the gitignore as well...

**Comment by ghost** [comment]
Sorry. Fixed :smile:

**Comment by jaytaph** [comment]
Thanks.. will try and merge this later today..

**Comment by jaytaph** [comment]
Thanks @steazzalini

**Pull request merged**